tests/scripts/create-dev-cluster.sh

```diff
@@ -18,23 +18,37 @@ wait_for_ceph_cluster() {
 }
 
 get_minikube_driver() {
+	local DRIVER
+	if [[ -n "${minikube_driver}" ]];  then
+		DRIVER="${minikube_driver}"
+	fi
```

Comment:
see more comments below, unless there's a real use case for this I don't see the need for the new option `-y`

Reply:
@rkachach  wrote:

> see more comments below, unless there's a real use case for this I don't see the need for the new option `-y`

I need it because the auto-detected driver `hyperkit` does not work for me on an Intel Mac, even if install hyperkit from brew. `qemu` works though. That's why I need to be able to override

Reply:
Shouldn't the script auto-select qemu if it detects the arm architecture?